action.yml

```
@@ -3,8 +3,10 @@ description: 'A GitHub Action to deploy an artifact to GitHub Pages'
 runs:
   using: 'node16'
   main: 'dist/index.js'
-  pre: 'pre/index.js'
 inputs:
+  emit_telemetry:
+    description: 'Should this action emit build telemetry instead of deploy the build artifact'
```

Small adjustment to the phrasing here:

```suggestion
    description: 'Should this action only emit build telemetry instead of deploying the build artifact?'
```